Test/ValidationErrorsConstraint.php

```diff
@@ -11,9 +11,13 @@
 
 namespace Liip\FunctionalTestBundle\Test;
 
+// BC
+class_alias('\PHPUnit_Framework_Constraint', '\PHPUnit\Framework\Constraint\Constraint');
```

Comment:
@HeahDude This may be a stupid question but is it still necessary after your last commit that require PHPUnit `^4.8.35|…`?